Resources/Locale/en-US/crew-medals/crew-medal-component.ftl

```diff
@@ -0,0 +1,20 @@
+# interaction
+comp-crew-medal-inspection-text = Awarded to {$recipient} for {$reason}.
+comp-crew-medal-award-text = {$recipient} has been awarded the {$medal}.
+
+# round end screen
+
+comp-crew-medal-round-end-result = {$count ->
+    [one] There was one medal awarded:
+    *[other] There were {$count} medals awarded:
+}
+
+comp-crew-medal-round-end-list = [color=white]{$recipient}[/color] earned the [color=white]{$medal}[/color] for
```

Comment:
reason definitely should be moved here because different languages can use it different
also i suggest to write here who awarded the recipient

Reply:
Good point, I can move it in there if I can figure out how to handle the indentation for the list.
I tried adding the person who awarded the medal, but the events for equipping/unequipping are a terrible mess and don't include that information.

Reply:
The localization suggestion is done.